File: app/src/main/java/com/kelsos/mbrc/ui/activities/HelpFeedbackActivity.java
package com.kelsos.mbrc.ui.activities;

import android.os.Bundle;
import android.support.design.widget.TabLayout;
import android.support.v4.view.ViewPager;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import butterknife.BindView;
import butterknife.ButterKnife;
import com.kelsos.mbrc.R;
import com.kelsos.mbrc.adapters.HelpFeedbackPagerAdapter;

public class HelpFeedbackActivity extends AppCompatActivity {

  @BindView(R.id.toolbar)
  Toolbar toolbar;
  @BindView(R.id.feedback_tab_layout)
  TabLayout tabLayout;
  @BindView(R.id.pager_help_feedback)
  ViewPager viewPager;

  private HelpFeedbackPagerAdapter pagerAdapter;

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_help_feedback);
    ButterKnife.bind(this);
    setSupportActionBar(toolbar);
    ActionBar actionBar = getSupportActionBar();

    if (actionBar != null) {
      actionBar.setDisplayHomeAsUpEnabled(true);
      actionBar.setHomeButtonEnabled(true);
    }

    pagerAdapter = new HelpFeedbackPagerAdapter(getSupportFragmentManager(), this);
    viewPager.setAdapter(pagerAdapter);
    tabLayout.setupWithViewPager(viewPager);
  }
}

File: app/src/main/kotlin/com/kelsos/mbrc/ui/help_feedback/HelpFeedbackActivity.kt
package com.kelsos.mbrc.ui.help_feedback

import android.os.Bundle
import android.support.design.widget.TabLayout
import android.support.v4.view.ViewPager
import android.support.v7.widget.Toolbar
import com.kelsos.mbrc.R
import com.kelsos.mbrc.ui.activities.FontActivity
import kotterknife.bindView

class HelpFeedbackActivity : FontActivity() {

  private val toolbar: Toolbar by bindView(R.id.toolbar)
  private val tabLayout: TabLayout by bindView(R.id.feedback_tab_layout)
  private val viewPager: ViewPager by bindView(R.id.pager_help_feedback)

  private var pagerAdapter: HelpFeedbackPagerAdapter? = null

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_help_feedback)

    setSupportActionBar(toolbar)
    val actionBar = supportActionBar

    if (actionBar != null) {
      actionBar.setDisplayHomeAsUpEnabled(true)
      actionBar.setHomeButtonEnabled(true)
    }

    pagerAdapter = HelpFeedbackPagerAdapter(supportFragmentManager, this)
    viewPager.adapter = pagerAdapter
    tabLayout.setupWithViewPager(viewPager)
  }
}
